Apromore-Core-Components/Apromore-Manager/src/test/java/org/apromore/builder/UserManagementBuilder.java

```diff
@@ -87,15 +104,15 @@ public UserManagementBuilder withUserMetaDataType(String type, int version, bool
 	        return this;
 
 	}
-	
+
```

Comment:
We probably need to choose between Usermetadata or UserMetadata or UserMetaData.